ui/src/page/title.rs

```diff
@@ -0,0 +1,35 @@
+use crate::view::element::{text, Element};
+use crate::view::grid::cell::cell;
+use crate::view::grid::row;
+use crate::view::grid::row::row;
+
+////////////////////////////////////////////////////////////////
+// Types  //
+////////////////////////////////////////////////////////////////
+
+#[derive(Clone, Copy)]
+pub struct Model {}
+
+pub enum Msg {
+    StartClicked,
+}
+
+////////////////////////////////////////////////////////////////
+// Init //
+////////////////////////////////////////////////////////////////
+
+pub fn init() -> Model {
+    Model {}
+}
+
+////////////////////////////////////////////////////////////////
+// View //
+////////////////////////////////////////////////////////////////
+
+pub fn view(_model: Model) -> Vec<Element<Msg>> {
+    row::many(vec![
+        row(vec![cell(vec![text("FIGHT LINES")])]).center(),
+        row(vec![cell(vec![text("THE GAME")])]).center(),
+    ])
```

Comment:
It looks kind of like this
```
                           FIGHT LINES                           |
                             THE GAME                            |
```
two rows, each with text on them, with their content centered. Thats what this code is building.

Reply:
Im ok with this